packages/sonarwhal/src/lib/cli/analyze.ts

```diff
@@ -63,6 +63,18 @@ const tryToLoadConfig = async (actions: CLIOptions): Promise<SonarwhalConfig> =>
     let config: SonarwhalConfig;
     const configPath: string = actions.config || SonarwhalConfig.getFilenameForDirectory(process.cwd());
 
+    if (!configPath) {
+        logger.error(`Couldn't find a valid path to load the configuration file.`)
+
+        const created = await askUserToCreateConfig();
+
+        if (created) {
+            config = await tryToLoadConfig(actions);
+        }
+
+        return config;
```

Comment:
Is this return ok? I mean, do we want to finish the process after create the config or we want to use that config to run sonarwhal?

Reply:
We want to use that config to run sonarwhal

Reply:
@sarvaje This return should be okay. tryToLoadConfig is only responsible for getting config but not running sonarwhal. Even though we return here, sonarwhal will still run later in the analyze function.

Reply:
@molant This is `return` in `tryToLoadConfig` but not `analyze`. So sonarwhal should still run after this.

Reply:
ahhh, I thought it was a return in `analyze`.